index.html

```diff
@@ -29,7 +29,103 @@ <h6 class="a">LET’S CONNECT</h6>
+        </div>
+      </section>
+      <section class=" Job-container" >
+        <nav class="img1">
+          <img src="images/SnapshootPortfolio1.png" alt="">
+        </nav>
+        <h1 class="tonic">Multi-Post Stories</h1>
+        <div>
+          <ul class="Jobs">
+            <li><h5> CANOPY</h5></li>
+            <li><img src="images/Counter.png" alt=""></li>
+            <li><h5>Back End Dev</h5></li>
+            <li><img src="images/Counter.png" alt=""></li>
+            <li><h5>2015</h5></li>
+          </ul>
+          <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="counter">
+            <li class="htmll"> <img src="images/Tag.png" alt=""> </li>
+            <li> <img src="images/Tag(1).png" alt=""> </li>
+            <li> <img src="images/Tag(2).png" alt=""> </li>
+          </ul>
+        </div>
+        <div class="Job1-btn">
+          <button type="button" name="button" class="Job1-button"> See Project</button>
+        </div>
+      </section>
+
+      <section class="container">
+        <h1 class="head">About <br> Myself </h1>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h6 class="a">LET’S CONNECT</h6>
+        <ul class="social-media-container">
+          <li class="i-angel"><a href="#"><img src="images/angellist.png" alt=""></a></li>
+          <li class="i-twitter"><a href="#"><img src="images/twitter.png" alt=""></a></li>
+          <li class="i-github"><a href="#"><img src="images/github.png" alt=""></a></li>
+          <li class="i-medium"><a href="#"><img src="images/medium.png" alt=""></a></li>
+        </ul>
+        <div class="Job-btn">
+          <button type="button" name="button" class="Job2-button"> Get my resume</button>
+        </div>
+      </section>
+      <section class="Skills">
+        <div class="skill">
+          <ul class="lang">
+            <li><h2>Languages</h2></li>
+            <li><img src="images/Union.png" alt=""></li>
+          </ul>
+          <ul class="skills">
+            <li class="sk">
+              <img src="images/Ellipse.png" alt="">
+              <h4>JavaSript</h4>
+            </li>
+            <li class="sk">
+              <img src="images/Ellipse1(1).png" alt="">
+              <h4>HTML</h4>
+            </li>
+            <li class="sk">
+              <img src="images/Ellipse1(2).png" alt="">
+              <h4>CSS</h4>
+            </li>
+          </ul>
+        </div>
+
+
+
 
+        <div class="lang1">
+          <h2>Frameworks</h2>
+          <img src="images/Union1.png" alt="">
+        </div>
+        <div class="lang1">
+          <h2>Skills</h2>
+          <img src="images/Union1.png" alt="">
```

Comment:
Please use `ul` and `li` for `Languages`, `Frameworks`, and `Skills` as stated in the project requirement.

<img width="953" alt="image" src="https://user-images.githubusercontent.com/34578631/171066532-455915f1-7647-4311-ba4f-edf61625c6e2.png">

And change the text font and center the section to match the Figma design.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="408" alt="image" src="https://user-images.githubusercontent.com/34578631/171066680-aaa6ce3b-bd91-443a-93fc-526c2db8041d.png"> | <img width="373" alt="image" src="https://user-images.githubusercontent.com/34578631/171066690-6d7877aa-3629-4977-bd59-9c47e88177fc.png"> |